slices/gearman-job-server.yaml

```diff
@@ -0,0 +1,34 @@
+package: gearman-job-server
+
+essential:
+  - gearman-job-server_copyright
+
+slices:
+  bins:
+    contents: 
+      /usr/sbin/gearmand:
+
+  libs:
+    # gearman-job-server actually depends on adduser as well to create and
+    # remove a user during package install/removal - however we don't 
+    # support this currently so adduser is not added here.
+    essential:
+      - libboost-program-options1.83.0_libs
+      - libc6_libs
+      - libevent-2.1-7t64_libs
+      - libevent-pthreads-2.1-7t64_libs
+      - libgcc-s1_libs
+      - libhiredis1.1.0_libs
+      - libmemcached11t64_libs
+      - libmysqlclient21_libs
+      - libpq5_libs
+      - libsqlite3-0_libs
+      - libstdc++6_libs
```

Comment:
There's no need for a `libs` slice for gearman-job-server.
Instead you can put the essential part under the `bins` slice, like this:
```
  bins:
    essential:
      - libboost-program-options1.83.0_libs
      - libc6_libs
      - libevent-2.1-7t64_libs
      - libevent-pthreads-2.1-7t64_libs
      - libgcc-s1_libs
      - libhiredis1.1.0_libs
      - libmemcached11t64_libs
      - libmysqlclient21_libs
      - libpq5_libs
      - libsqlite3-0_libs
      - libstdc++6_libs
    contents: 
      /usr/sbin/gearmand:
```